### PR TITLE
[NO GBP] Donator Items - Commanding and Comfy Plushies

### PR DESCRIPTION
**Pull request by kannthus**
## About The Pull Request

Requested in a discord ticket by 2 donators:
- 2 plushies added, restricted to their ckeys.

## How This Contributes To The Skyrat Roleplay Experience

N/A

### CODE REVIEW

File: modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm [hunk 1]
```diff
@@ -495,6 +495,17 @@
 	item_path = /obj/item/toy/plush/skyrat/chunko/inessa
 	donator_only = TRUE
 
+
+/datum/loadout_item/toys/plushie_chiara
+	name = "Chiara Plushie"
+	item_path = /obj/item/toy/plush/skyrat/plushie_chiara
+	ckeywhitelist = list("SRQ","Superlagg")
```
Comment (jjpark-kb) [inline modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm]: all lowercase

File: modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm [hunk 2]
```diff
@@ -495,6 +495,17 @@
...
+
+/datum/loadout_item/toys/plushie_dan
+	name = "Dan Plushie"
+	item_path = /obj/item/toy/plush/skyrat/plushie_dan
+	ckeywhitelist = list("SRQ","Superlagg")
```
Comment (jjpark-kb) [inline modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm]: all lowercase

**Review by tf-4 — CHANGES_REQUESTED** [review]
Don't use their character names on anything player-facing, please.

**Comment by kannthus** [comment]
> Don't use their character names on anything player-facing, please.

Could you elaborate on player-facing please, as the plushies themselves are technically player-facing. ~~Changing the names as we speak though~~ Done.

**Comment by tf-4** [comment]
> > Don't use their character names on anything player-facing, please.
> 
> Could you elaborate on player-facing please, as the plushies themselves are technically player-facing. ~Changing the names as we speak though~ Done.

Anything that isn't an icon state or item path, pretty much. So you're fine there

File: modular_skyrat/modules/customization/game/objects/items/plushes.dm [hunk 3]
```diff
@@ -572,3 +572,16 @@
 	icon_state = "plushie_star"
 	gender = FEMALE
 	squeak_override = list('modular_skyrat/modules/emotes/sound/voice/trills.ogg' = 1)
+
```
Comment (Zonespace27) [inline modular_skyrat/modules/customization/game/objects/items/plushes.dm:575]:
```suggestion
```

File: modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm [hunk 4]
```diff
@@ -495,6 +495,17 @@
 	item_path = /obj/item/toy/plush/skyrat/chunko/inessa
 	donator_only = TRUE
 
+
```
Comment (Zonespace27) [inline modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm]:
```suggestion
```

File: modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm [hunk 5]
```diff
@@ -495,6 +495,17 @@
...
+	name = "Commanding Fox Plushie"
+	item_path = /obj/item/toy/plush/skyrat/plushie_chiara
+	ckeywhitelist = list("srq","superlagg")
+
+/datum/loadout_item/toys/plushie_dan
+	name = "Comfy Fox Plushie"
+	item_path = /obj/item/toy/plush/skyrat/plushie_dan
+	ckeywhitelist = list("srq","superlagg")
+
 //Chunko Fops were donated by SlippyJoe, who requested they are usable by everyone
```
Comment (Zonespace27) [inline modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm]: this comment should be above the plushies

File: modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm [hunk 6]
```diff
@@ -495,6 +495,17 @@
 	item_path = /obj/item/toy/plush/skyrat/chunko/inessa
 	donator_only = TRUE
 
+
+/datum/loadout_item/toys/plushie_chiara
+	name = "Commanding Fox Plushie"
+	item_path = /obj/item/toy/plush/skyrat/plushie_chiara
+	ckeywhitelist = list("srq","superlagg")
```
Comment (Zonespace27) [inline modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm]:
```suggestion
	ckeywhitelist = list("srq", "superlagg")
```

File: modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm [hunk 7]
```diff
@@ -495,6 +495,17 @@
...
+
+/datum/loadout_item/toys/plushie_dan
+	name = "Comfy Fox Plushie"
+	item_path = /obj/item/toy/plush/skyrat/plushie_dan
+	ckeywhitelist = list("srq","superlagg")
```
Comment (Zonespace27) [inline modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm]:
```suggestion
	ckeywhitelist = list("srq", "superlagg")
```

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Please, no hard black outlines on sprites :( its grating on the eyes and makes it blend poorly with the environment

File: modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm (hunk 5 above)
Comment (kannthus) [reply modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm]: Which comment, The Chunko Fops one? As that's unrelated.

Comment (kannthus) [reply modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm]: Ah I see what you mean now, moved it up

**Pull request merged**